### ISSUE TITLE
Angular 4 support

### ISSUE THREAD

**Issue by elhoo** [zefoy/ngx-voog-wysihtml#4]
Hi,

Is there any plans for supporting angular 4? At the moment the component works locally but when I build for deploy I get the following error:
`[ERROR] Unexpected value 'WysiHtmlModule in /Users/lh/dev/projects/recipe4/node_modules/angular2-voog-wysihtml/lib/src/wysihtml.module.d.ts' imported by the module 'Recipe4RecipeModule in /Users/lh/dev/projects/recipe4/src/main/webapp/app/entities/recipe/r
ecipe.module.ts'. Please add a @NgModule annotation.`

Which I think is related to http://stackoverflow.com/questions/43210804/issues-with-ngmodule-when-using-angular-4

**Comment by sconix** [comment]
Yes there is. We should start using this library in our main project during the summer. So most likely during June or at the latest on July we will work on this library.

**Comment by sconix** [comment]
I will try to find time to bring the Angular 4 support after couple weeks the other planned improvements will most likely go to July.

**Comment by jmls** [comment]
did you ever get to do an angular4 port ?

**Comment by sconix** [comment]
We are starting to be in the point where we need this sort of component in our app. So we are going to do an evaluation soon if this is still the best option or if there is something better. Most likely we will use something else like medium-editor.
